### PR TITLE
Added Apply Pipe

### PR DESCRIPTION
**Pull request by dxArtemiusz**
Pipe is based on the [help topic](https://js.devexpress.com/Documentation/Guide/Angular_Components/Component_Configuration_Syntax/#Angular_Change_Detection_Specifics)

### CODE REVIEW

File: JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts [hunk 1]
```diff
@@ -32,88 +43,75 @@ export class AppComponent {
     });
   }
 
-  onOptionChanged(e: any) {
+  onOptionChanged = (e: any) => {
```
Comment (williamvinogradov) [inline JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts]: Minor comment.

Can we improve typing here and replace an ```any``` with a strict type?

File: JSDemos/Demos/Scheduler/CurrentTimeIndicator/Angular/app/app.component.html [hunk 2]
```diff
@@ -17,12 +17,14 @@
...
-      <img [src]="getMovieById(model.appointmentData.movieId).image" />
-      <div class="movie-text">{{
-        getMovieById(model.appointmentData.movieId).text
-      }}</div>
-    </div>
+    <ng-container
+      *ngIf="getMovieById | apply : model.appointmentData.movieId as movie"
+    >
+      <div class="movie">
+        <img [src]="getMovieById(model.appointmentData.movieId).image" />
```
Comment (williamvinogradov) [inline JSDemos/Demos/Scheduler/CurrentTimeIndicator/Angular/app/app.component.html]: Can we use the ```movie``` template variable here?

```suggestion
        <img [src]="movie.image" />
```

File: JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts (hunk 1 above)
Comment (dxArtemiusz) [reply JSDemos/Demos/Scheduler/CellTemplates/Angular/app/app.component.ts]: Changed all any in this file to scheduler event types

File: JSDemos/Demos/Scheduler/CurrentTimeIndicator/Angular/app/app.component.html (hunk 2 above)
Comment (dxArtemiusz) [reply JSDemos/Demos/Scheduler/CurrentTimeIndicator/Angular/app/app.component.html]: Changed it

**Review by williamvinogradov — APPROVED** [review]

**Pull request merged**